### PR TITLE
[Merged by Bors] - chore(Data/Int/Defs): clear out Init/Data/Int/Basic

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
The build fails, but looks good to me apart from that. Thanks for clearing this out!
maintainer delegate (i.e., feel free to maintainer merge on my behalf if the build only requires trivial fixes)

File: Mathlib/Tactic/Lift.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
 -/
 import Mathlib.Tactic.Basic
-import Mathlib.Init.Data.Int.Basic
+import Mathlib.Data.Int.Defs
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/Lift.lean:7]: This one is circular

Comment (jcommelin) [reply Mathlib/Tactic/Lift.lean:7]: Let's see if we can just remove it
```suggestion
```

**Comment by grunweg** [comment]
CI is green - looks good to me!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**